pom.xml

```diff
@@ -31,7 +31,7 @@
     </distributionManagement>
 
     <properties>
-        <dependency.alfresco-core.version>6.14</dependency.alfresco-core.version>
```

Comment:
Why downgrade?
This implies a broken core version that can't be used.
This pom file change should be a separate commit.